src/components/blocks/image.ts

```diff
@@ -16,20 +20,38 @@ interface Image {
 
 // ----- Functions ----- //
 
+const imageRatioStyles = (sizes = "100vw", width: number, height: number): SerializedStyles => css`
+        --size: ${sizes};
+        height: calc(var(--size) * ${height / width});
```

Comment:
Out of curiosity, what is the CSS variable used for here?

Reply:
It was a way to use a `calc` inside a `calc`. Might be able to refactor it some way

Reply:
🤔 where is there a `calc` inside a `calc?

Reply:
`sizes` can be a calc :)